site/en/getstarted/cluster/install_cluster-milvusoperator.md

```diff
@@ -103,7 +103,7 @@ mutatingwebhookconfiguration.admissionregistration.k8s.io/milvus-operator-mutati
 validatingwebhookconfiguration.admissionregistration.k8s.io/milvus-operator-validating-webhook-configuration created
 ```
 
-### Check milvus-operator status
+## Check milvus-operator status
 
 Run `$ kubectl get -n milvus-operator deploy/milvus-operator` to check if Milvus Operator is running. If so, you can see the Milvus Operator deploy running as shown in the following output.
```

Comment:
Run `$ kubectl get -n milvus-operator deploy/milvus-operator` to check if Milvus Operator is running. You can see the following output is Milvus Operator is running.